.github/pull_request_template.md

```diff
@@ -1,4 +1,4 @@
-#### DESCRIPTION
+#### DESCRIPTION (include screenshots, and mobile screenshots for participant UX)
```

Comment:
Should we make this a whole new section? at least for me I would probably skip over it mentally but if it was a full section under description I would be less likely to skip the screenshots part. I can go either way tho so I'm gonna go ahead and approve.

Reply:
yeah, I was torn.  I don't like the idea of a separate screenshot section because I like it best when screenshots are integrated into the description.  But agree it could get glossed over.  Let's see how it works